release/local.sh

```diff
@@ -20,21 +20,16 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 stage "Content Provider - Local Folder"
```

Comment:
About commit msg: if its an upstream or a downstream should not matter. Even why an upstream should not be using local rpm injection. That feature is more for development and custom builds.